kano_qt_apps/lib/kano_apps_deps.pri

```diff
@@ -1,3 +1,14 @@
+exists(conanbuildinfo.pri) {
```

Comment:
Ah, using Conan? :)

Reply:
Trying it out but it is purely optional.